libgap.pc.in

```diff
@@ -0,0 +1,12 @@
+prefix=@prefix@
+exec_prefix=@exec_prefix@
+libdir=@libdir@
+includedir=@prefix@/include
+
+Name: GAP
+Description: Library containing the kernel of GAP, a computer algebra system
+URL: https://www.gap-system.org
+Version: @PACKAGE_VERSION@
+Requires.private: gmp
```

Comment:
wait, doesn't it also require `zlib`, and possibly `readline`, depending on the install options? Otherwise static linking won't work after all, will it?

Did you test this at all? Static linking libgmp doesn't strike me as very useful, so perhaps this line should just be removed?

I mean, I am also fine with keeping it, and won't block this PR over it; but it seems misleading?

Reply:
what, for libgap? Does libgap depend on zlib and readline?

Reply:
Sure, we always link against libz. Whether we link against GNU readline depends on `configure`.

Reply:
perhaps, we should just remove `Requires` line, and wait for possible bug reports (from some clever Linux distro which likes to underlink stuff...)